app/packs/src/apps/mydb/inbox/UnsortedDataset.js

```diff
@@ -145,10 +135,9 @@ export default class UnsortedDataset extends React.Component {
   dropzone() {
     return (
       <Dropzone
-        onDrop={files => this.handleFileDrop(files)}
-        style={{ height: 50, width: '100%', border: '3px dashed lightgray' }}
-      >
-        <div style={{ textAlign: 'center', paddingTop: 12, color: 'gray' }}>
+        onDrop={(files) => this.handleFileDrop(files)}
+        className="border-dashed border-gray-200">
```

Comment:
The drop zone does not have a 3px border anymore
Is it correct?
Haven't you uses `border-3` at other drop zones?

Reply:
Yes, I will make it border-3 to be consistent :)